src/Text/Pandoc/ImageSize.hs

```diff
@@ -116,7 +116,11 @@ imageType img = case B.take 4 img of
                      _                  -> mzero
 
 findSvgTag :: ByteString -> Bool
-findSvgTag img = B.null $ snd (B.breakSubstring img "<svg")
+findSvgTag img = prfx `elem` ["<svg ", "<svg\n", "<svg\r"]
+  where
+    prfx = B.take 5 $ last $ B.groupBy openingTag $ B.drop 7 img
+    -- B.groupBy openingTag matches first "<svg" or "<html" but not "<!--"
+    openingTag x y = x == '<' && y /= '!'
```

Comment:
@mb21,  adding commands and checking if the svg will be parsed, seems to be rather difficult. For example, the following command does not verify the image size (same for `-t html`):

```
% pandoc -f latex -t opendocument --default-image-extension=svg
\includegraphics{../svg-test.svg}
^D
<text:p text:style-name="Text_20_body"><draw:frame draw:name="img1"><draw:image xlink:href="../svg-inkscape.0.92" xlink:type="simple" xlink:show="embed" xlink:actuate="onLoad" /></draw:frame></text:p>
```

If I use the option `-t odt`, I need to specify an output file. Verify the image size?